File: Python/nieparzyste.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  nieparzyte.py

import random

def losuj_nieparzyste(maks):
    for i in range(200):
        liczba = random.randint(0, MAKS_LICZBA)
        if liczba % 2:
            nieparzyste.append(liczba)
    print(len(nieparzyste))
    
def main(args):
    MAKS_LICZBA = 200
    liczby = []
    suma = 0
    licznik = 0
    
    for i in range(200):
        liczby.append(random.randint(0, MAKS_LICZBA))
   
    for liczba in liczby:
        if liczba % 2:
            suma = suma + liczba
            licznik += 1
            
    print("suma wylosowanych liczb: ", suma, "liczba wylosowanych liczb: ", licznik)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/nieparzyste.cpp
/*
 * nieparzyste.cpp
 */


#include <iostream>

using namespace std;

int wypisz(int n) {
    int i = 1;
    for(i =! n; i++){
        i = i + 2;
    }
    return i;
}


int main(int argc, char **argv)
{
	int n;
    cout << "Podaj dowolną liczbę naturalną:";
    cin >> n;
    cout << "liczby: " << wypisz(n) << endl;
	return 0;
}

